src/components/network/selection-creation-panel.tsx

```diff
@@ -274,21 +281,13 @@ const SelectionCreationPanel: React.FC<SelectionCreationPanelProps> = ({
                     <Button
                         variant="outlined"
                         type={'submit'}
-                        disabled={!formMethods.formState.isDirty || savingState}
-                        onClick={() => {
-                            formMethods.trigger().then((isValid) => {
-                                if (isValid && defaultFolder) {
-                                    onSaveSelection(
-                                        formMethods.getValues() as ISelectionCreation,
-                                        defaultFolder,
-                                        setSavingState
-                                    );
-                                }
-                            });
-                        }}
+                        disabled={
```

Comment:
we do not use isValid, it's isDirty

Reply:
i have some strange behavior with isDirty.
i suggest to keep it as it is , because the behavior of this button was like that, i just moved the code